### PR TITLE
LGA-3343 - Add care in the community category

### PR DESCRIPTION
**Pull request by said-moj**
## What does this pull request do?

Add care in the community category.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Review by TawneeOwl — APPROVED** [review]
LGTM

File: app/templates/categories/community_care/landing.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+             _("Problems if the local authority is not providing the right support. Includes problems with housing, financial support, education, personal advisers and Pathway Plans."),
+             url_for("categories.community_care.care_leaver")) }}
+
+        <br>
+
+        <h2 class="govuk-heading-m">{%  trans%}More problems{% endtrans %}</h2>
+
+        <br>
+
+            {{ cannot_find_your_problem("community care", url_for("categories.community_care.other"))}}
```
Comment (BenMillar-MOJ) [inline app/templates/categories/community_care/landing.html:60]: Should this have an underscore, for the localisation?

File: app/templates/categories/community_care/landing.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+             _("Issues with safeguarding, abuse, neglect, or care that is not good enough. This includes in group homes or other placements, or from a carer or social worker."),
+             url_for("categories.community_care.problems_with_quality_of_care")) }}
+
+        {{ list_item(_("If you’re a care leaver"),
+             _("Problems if the local authority is not providing the right support. Includes problems with housing, financial support, education, personal advisers and Pathway Plans."),
+             url_for("categories.community_care.care_leaver")) }}
+
+        <br>
+
+        <h2 class="govuk-heading-m">{%  trans%}More problems{% endtrans %}</h2>
```
Comment (BenMillar-MOJ) [inline app/templates/categories/community_care/landing.html:56]: The spacing is off around this `{% trans %}` block opening

**Review by BenMillar-MOJ — CHANGES_REQUESTED** [review]

File: app/templates/categories/community_care/landing.html (hunk 2 above)
Comment (said-moj) [reply app/templates/categories/community_care/landing.html]: Fixed spacing

**Review by BenMillar-MOJ — APPROVED** [review]

**Pull request merged**